test/cppsuite/test_harness/random_generator.h

```diff
@@ -0,0 +1,54 @@
+/* Include guard. */
+#ifndef RANDOM_GENERATOR_H
+#define RANDOM_GENERATOR_H
+
+#include <algorithm>
+#include <cstddef>
+#include <random>
+#include <string>
+
+namespace test_harness {
+
+/* Generate random values. */
```

Comment:
This comment could be worded better, something like "Helper class to generate random values".